scripts/download.pl

```diff
@@ -82,8 +82,8 @@ ($)
 	}
 
 	return $have_curl
-		? (qw(curl -f --connect-timeout 20 --retry 5 --location --insecure), shellwords($ENV{CURL_OPTIONS} || ''), $url)
```

Comment:
I think it should be configurable (with checking certs by default). sometimes certs are not updated by ops/scripts and you may want to download without cert check and rely on checksums